server/views/premises/bookings/new.njk

```diff
@@ -11,33 +12,38 @@
 
 {% block content %}
 
-    <h1>Make a booking</h1>
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-            <form action="/premises/{{premisesId}}/bookings" method="post">
-                <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+	<h1>Make a booking</h1>
```

Comment:
I think we have different formatters set for `.njk` files, I don't mind which one we used but would be align. 
I use "okitavera.vscode-nunjucks-formatter"

Reply:
Cool - I'll stick with yours - I was using a Twig formatter that wasn't great, but the one you use looks much better!